src/Agent.Listener/Configuration/CredentialManager.cs

```diff
@@ -18,7 +18,8 @@ public class CredentialManager : AgentService, ICredentialManager
         public static readonly Dictionary<string, Type> CredentialTypes = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase)
         {
             { "PAT", typeof(PersonalAccessToken)},
-            { "ALT", typeof(AlternateCredential)}
+            { "ALT", typeof(AlternateCredential)},
```

Comment:
Yeah, the ALT credentials are a separate thing. I noticed ALT does not work, because some of the validation code was copied from the PAT credentials.